src/applications/pact-act/sass/pact-act.scss

```diff
@@ -1,4 +1,7 @@
+  /* stylelint-disable */
```

Comment:
What was this added for?

Reply:
This was added since style lint was throwing an error for the hex code. Couldn't find a way to fix it and it looked fine to me. Let me know if you have a fix with using the hex code.

Reply:
What's the lint error?

Reply:
I was getting this error https://stylelint.io/user-guide/rules/color-no-hex/

Reply:
Is there a color in the [color palette](https://design.va.gov/foundation/color-palette) we can use on the element instead of adding it via CSS? I think you'd have to replace `$color-gray-dark` with `vads-u-color--gray-dark` or something like that. If we don't have a match, a hex is fine.

Reply:
Thanks! I was able to get the color that the DS uses.